ISSUE_TRIAGE_PROCESS.rst

```diff
@@ -30,6 +30,52 @@ to fix an issue or make an enhancement, without needing to open an issue first.
 This is intended to make it as easy as possible to contribute to
 the project.
 
+Another important part of our Issue reporting process are also Github Discussions.
+Issues should represent clear feature requests or bugs which can/should be either implemented or fixed.
+Users are encouraged to open Discussions rather than Issues if there are no clear, reproducible
+steps, or when they have troubleshooting problems.
+
+Responding to issues/discussions (relatively) quickly
+'''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+It is vital to provide rather quick feedback to Issues and Discussions opened by our users, so that they
+feel listened to rather than ignored. Even if the response is "we are not going to work on it because ...",
+or "converting this issue to discussion because ..." or "closing because it is a duplicate of #xxx", it is
```

Comment:
Not directly. We do not have anyone looking at the backlog and we will not have likely. We smply leave issues open - if we have not enough information, we ask for it in the issue and that's basically it. Since all issues are open and visible to anyone (and searchable) if other people will see it, they might add extra information that might lead us to further investigations. 

However we have something "like" that. If we think that issue is important but we have no clue we often add it to "next bugfix milestone". Which is an indication that we at least should look at it before the next release and we can decide during release preparation what we should do with it. 

That's cool you mentioned it - this was something done implicitly without actually writing it down. I will add it here.

Reply:
This was helpful explanation. Yea, I read about "next bugfix milestone" in slack conversation, but didn't see it in the GitHub rst. Thanks for adding it.